### PR TITLE
#167979751 user can create a petition

### PR DESCRIPTION
**Pull request by gik-hub**
### What does this PR do?
This PR adds functionality to ensure that a user can create a petition
#### Description of Task to be completed?
Add Petitioner service to link to the database, a petitioner controller and routes
#### How should this be manually tested?
Review the source code
#### Any background context you want to provide?
This work is still in progress and not has been made for other members to pick up the project flow
#### What are the relevant pivotal tracker stories?
[#167979751](https://www.pivotaltracker.com/story/show/167979751)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/controllers/PetitionController.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        const votes = 0;
+        const expired = false;
+        const { title, description } = req.body;
+        let user_id = 1; // get from the token
+        const newPetition = {
+            user_id, title, description, votes, expired
+        }
+        try {
+            const createdPetition = await PetitonService.addPetition(newPetition);
+            console.log('-----', createdPetition);
```
Comment (Jordybastien) [inline src/controllers/PetitionController.js:17]: Remove logs

File: src/services/PetitionServices.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import database from '../models';
+
+class PetitonService {
+    static async addPetition(newPetiton) {
+        // console.log(Object.keys(database.petitions));
```
Comment (Jordybastien) [inline src/services/PetitionServices.js:5]: Remove these commented out codes

**Review by Jordybastien — APPROVED** [review]
Apart from those left out logs, the rest looks good

**Review by mucyomiller — APPROVED** [review]
LGTM

**Review by amilykassim — APPROVED** [review]
Thanks for adding those backslashes on all those endpoints as it was impeding the routes to work as expected, everything is ok by the way.

**Review by didasmbalanya — CHANGES_REQUESTED** [review]
Remove (console.log)log messages before merging with develop

**Review by didasmbalanya — APPROVED** [review]

**Comment by mucyomiller** [comment]
PR closed after a merge conflict was resolved by command-line tools.

**Pull request closed without merging**